tests/cases/fourslash/refactorInferFunctionReturnType23.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path='fourslash.ts' />
+
+////const foo = async /*a*//*b*/a => {

```

Comment:
Is this a duplicate of `refactorInferFunctionReturnType23.ts`?

Reply:
@jessetrinity Yes, you are right. It seems the conflicts were not resolved correctly with https://github.com/microsoft/TypeScript/pull/42737. Thanks for pointing that out. ;)